ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/arp.j2

```diff
@@ -1,7 +1,7 @@
 {# eos - ARP settings #}
-{% if arp is defined and arp is not none %}
-{%     if arp.aging is defined and arp.aging is not none %}
-{%         if arp.aging.timeout_default is defined and arp.aging.timeout_default is not none %}
+{% if arp is arista.avd.defined %}
```

Comment:
As` arista.avd.defined` does a deep test, this `if arp is arista.avd.defined` should be removed

Reply:
Same for the `arp.aging`